### PR TITLE
Reorganize structure and add npm commands

### PR DESCRIPTION
**Pull request by victorjzsun**
- Rename Test.js to index.js and remove old file
- Use forever package to run server in background, allowing users to chain server and client creation commands

### CODE REVIEW

**Review by Yuxuan-Lin — APPROVED** [review]
I have reviewed all the codes and they seem nice! Only two questions after approval: 1. Is it ok not having the 404 error alert? Or it just doesn't matter 2. What does "npm run server" do? Do we use that instead of "run start" later?

**Comment by victorjzsun** [comment]
1. All I did was delete index.js and rename Test.js to index.js. I didn't remove that on purpose, we just never had that piece in the express (Test.js) version.

2. `npm run server` runs the API server. `npm run start` runs the client-side web app. `npm run app` runs both the API and client, which is what you use from now on to run everything. I made a change to the README to reflect this change

PS: Usually you only approve when you have no more comments, if you want to add a comment, use the comment field as shown below.
image

**Comment by Yuxuan-Lin** [comment]
OK. Thanks!

**Pull request merged**
